pages/_app.js

```diff
@@ -1,10 +1,32 @@
 import GlobalStyle from "../styles";
+import plants from "@/lib/data";
+import { useState } from "react";
+import Image from "next/image";
+import styled from "styled-components";
 
 export default function App({ Component, pageProps }) {
+  const [initialPlants, setInitialPlants] = useState(plants);
```

Comment:
you don't need a state yet, also you don't need to place it in the _app file yet. Please keep your code as simple as possible at all times.

Reply:
Hey @F-Kirchhoff, thanks for your feedback. We actually considered initializing the data in data.js as a const variable in homepage.js and discussed that. However, we decided to implement the state variable in this user story as we will need it in the upcoming user stories in the first sprint already. That's why we defined the tasks like that which have been approved. Therefore, we would like to stay with the state here :)

Reply:
Hey Carmen,
I know this is quite picky, but it highlights a problem a lot of beginners face: exponential complexity in their codebase. This is a very common pattern - I will need it anyways in the future so I build it in now. But the reality is, that you don't know if you need it. This added complexity rarely gets removed and you end up with a lot of complex code that is simply not needed.

It might well be that this case is an exception, but I would like you to still get into "good habits". And especially in the habit of "code for the moment". Therefore please for now just import the plants directly in your homepage and use it without a state.